plugins/security/resources/jenkins.groovy

```diff
@@ -22,10 +22,23 @@ import hudson.model.View
 import hudson.security.Permission
 import jenkins.model.Jenkins
 import com.cloudbees.plugins.credentials.CredentialsProvider
-import com.sonyericsson.hudson.plugins.gerrit.trigger.PluginImpl
 import hudson.scm.SCM
 
 
+/*
+ * Optional imports
+ */
+
+GroovyClassLoader loader = Actions.class.getClassLoader();
+
+try {
+  GerritTriggerPluginImpl = loader.loadClass("com.sonyericsson.hudson.plugins.gerrit.trigger.PluginImpl")
```

Comment:
It is now defined but equals to None.

This will lead to exception if someone adds gerrit-related options in jenkins.yaml and applies this yaml on Jenkins without Gerrit. Which I think is a good thing